pkg/login/social/github_oauth.go

```diff
@@ -190,23 +190,31 @@ func (s *SocialGithub) UserInfo(client *http.Client, token *oauth2.Token) (*Basi
 		return nil, fmt.Errorf("error getting user info: %s", err)
 	}
 
-	err = json.Unmarshal(response.Body, &data)
-	if err != nil {
-		return nil, fmt.Errorf("Error getting user info: %s", err)
+	if err = json.Unmarshal(response.Body, &data); err != nil {
+		return nil, fmt.Errorf("error unmarshalling user info: %s", err)
 	}
 
 	teamMemberships, err := s.FetchTeamMemberships(client)
 	if err != nil {
-		return nil, fmt.Errorf("Error getting user teams: %s", err)
+		return nil, fmt.Errorf("error getting user teams: %s", err)
 	}
 
 	teams := convertToGroupList(teamMemberships)
 
+	role, err := s.extractRole(response.Body, teams)
+	if err != nil {
+		s.log.Error("Failed to extract role", "error", err)
+	}
+	if s.roleAttributeStrict && !role.IsValid() {
+		return nil, errors.New("invalid role")
```

Comment:
I did very minor mistake where my role was ending up as an empty role, and kept getting this error. Without looking at code and debugging, I would probably have hard time to figure out what is wrong. Would something like "could not resolve to the valid Grafana role from role_attribute_path parameter" help? Or what is better?